docker/Dockerfile.21.06

```diff
@@ -32,11 +32,19 @@ RUN apt-get update && apt-get install -y locales ninja-build && rm -rf /var/lib/
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
-RUN python3 setup.py install --use-cxx11-abi
+RUN MAX_JOBS=1 python3 setup.py install --use-cxx11-abi
 
 RUN conda init bash
 
-ENV LD_LIBRARY_PATH /opt/conda/lib/python3.8/site-packages/torch/lib:$LD_LIBRARY_PATH
+WORKDIR /opt/trtorch/src
+RUN mkdir -p /opt/trtorch/lib /opt/trtorch/bin
+RUN cp ./bazel-bin/cpp/bin/trtorchc/trtorchc /opt/trtorch/bin
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorch.so /opt/trtorch/lib
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorchrt.so /opt/trtorch/lib
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorch_plugins.so /opt/trtorch/lib
+
```

Comment:
This should be `./bazel-bin/cpp/lib/libtrtorch.so` and other libs as well, otherwise build fails

Reply:
Changed to using tarball - also installing to /usr/local